### ISSUE TITLE
main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []

### ISSUE THREAD

**Issue by derdeakon** [magento/magento2#2591]
Can anyone Help me with that log-entry ?

```
[2015-12-03 08:26:52] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 08:26:56] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 08:27:18] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 08:27:22] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 08:29:15] main.CRITICAL: Item (Magento\Catalog\Model\Product\Interceptor) with the same id "889" already exist [] []
[2015-12-03 08:29:34] main.CRITICAL: Item (Magento\Catalog\Model\Product\Interceptor) with the same id "551" already exist [] []
[2015-12-03 08:29:47] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 08:29:52] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 08:30:03] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 08:30:07] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2015-12-03 13:18:16] main.CRITICAL: Item (Magento\Catalog\Model\Product\Interceptor) with the same id "889" already exist [] []
```

**Comment by yaronish** [comment]
@derdeakon 
Thanks for reporting. Internal ticket: MAGETWO-47960

**Comment by andidhouse** [comment]
We have the same error log after using the migration tool from 1.7.0.2 to magento 2.0. All works ok but if we access a product in the product grid it just shows the headline of the product and nothing else. 

Maybe this is related to an extension we used in 1.7 and causes a wrong import in magento 2.0?

[2016-01-21 08:25:49] main.CRITICAL: Class media_image does not exist [] []
[2016-01-21 08:25:50] main.CRITICAL: Block product_tabs throws exception and cannot be rendered. [] []
[2016-01-21 08:25:50] main.CRITICAL: Block product_tabs throws exception and cannot be rendered. [] []
[2016-01-21 08:25:58] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2016-01-21 08:26:02] main.CRITICAL: Invalid template file: 'Magento_Catalog::js/components.phtml' in module: '' block's name: 'catalog_category_page_head_components' [] []
[2016-01-21 08:26:09] main.CRITICAL: Class media_image does not exist [] []
[2016-01-21 08:26:10] main.CRITICAL: Block product_tabs throws exception and cannot be rendered. [] []
[2016-01-21 08:26:10] main.CRITICAL: Block product_tabs throws exception and cannot be rendered. [] []

**Comment by mslabko** [comment]
Hello @derdeakon, issue was fixed for develop branch in 2d0775e215a98c14010fae67e85880a664dbe594
Thank you for reporting!
